e2e_tests/tests/experiment/test_native.py

```diff
@@ -228,3 +228,10 @@ def test_tf_estimator_warm_start(implementation: NativeImplementation, tf2: bool
 @pytest.mark.e2e_cpu  # type: ignore
 def test_pytorch_warm_start(implementation: NativeImplementation) -> None:
     run_warm_start_test(implementation)
+
+
+@pytest.mark.e2e_cpu
+def test_native_experiment_submit():
+    context_dir = conf.official_examples_path("trial/mnist_pytorch")
+    config = conf.load_config(conf.fixtures_path("mnist_pytorch/const-pytorch11.yaml"))
+    native_experiment_submit(context_dir, config, follow_logs=True)
```

Comment:
non-blocking: probably makes sens to add `experiment.wait_for_experiment_state()` to confirm it completes successfully.

Reply:
addressed in recent commit